packages/react-celo/src/components/qrcode.tsx

```diff
@@ -112,8 +112,8 @@ function matrixToDots(matrix: ReturnType<typeof generateMatrix>, size: number) {
         ) {
           dots.push(
             <circle
-              cx={i * cellSize + cellSize / 2}
```

Comment:
why?

Reply:
That has to be a leftover from playing around with it, to delete

Reply:
ah no it's a fix from the line 119 the radius of a cell is cellsize/3, so its center must be calculated properly.